vllm/model_executor/guided_decoding/guided_fields.py

```diff
@@ -0,0 +1,29 @@
+from dataclasses import dataclass
+from typing import Dict, List, Optional, Union
+
+from pydantic import BaseModel
+
+
+@dataclass
+class GuidedDecodingFields:
```

Comment:
Maybe rename this to `GuidedDecodingRequest` to be consistent with the other parameters to `LLM.generate`.